rct229/rulesets/ashrae9012019/section11/section11rule9.py

```diff
@@ -0,0 +1,71 @@
+from rct229.rule_engine.partial_rule_definition import PartialRuleDefinition
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_description
+from rct229.rulesets.ashrae9012019 import BASELINE_0
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_swh_bats_and_swh_use import (
+    get_swh_bats_and_swh_use,
+)
+from rct229.utils.utility_functions import find_exactly_one_service_water_heating_use
+
+
+class Section11Rule9(RuleDefinitionListIndexedBase):
+    """Rule 9 of ASHRAE 90.1-2019 Appendix G Section 11 (Service Water Heating)"""
+
+    def __init__(self):
+        super(Section11Rule9, self).__init__(
+            rmds_used=produce_ruleset_model_description(
+                USER=False, BASELINE_0=True, PROPOSED=True
```

Comment:
i thought this rule is only applicable to baseline? Should set USER and PROPOSED to False.

Reply:
Addressed.